app-accessibility/epos/epos-2.5.37-r2.ebuild

```diff
@@ -1,9 +1,9 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=5
+EAPI=6
 
-inherit eutils autotools
+inherit autotools
 
 DESCRIPTION="language independent text-to-speech system"
 HOMEPAGE="http://epos.ure.cas.cz/"
```

Comment:
homepage doesn't work

Reply:
Yeah, I think I may have noticed that in order to check what the software even is.
Not much I can do about that, however.

Reply:
https://sourceforge.net/projects/epos/
http://epos.ufe.cz/